### PR TITLE
Academyv2 changesv3

### PR DESCRIPTION
**Pull request by CryptoManion**
Fixing some articles / adding text / Resolving markdown issues

### CODE REVIEW

**Comment by CryptoManion** [comment]
Everything reviewed. Should be ready to go

File: docs/fundamentals/how-do-blockchains-work.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ _Nodes_ are operated by ‘miners’ who are individuals or entities that have a
 
 ![distributed ledger technology](/img/how-do-blockchains-work/distributed-ledger-technology.jpeg)
 
+### The Nakamoto Consensus
```
Comment (yurikoinaba) [inline docs/fundamentals/how-do-blockchains-work.md:97]: Do we intentionally have this section in 2 different places?

Comment (CryptoManion) [reply docs/fundamentals/how-do-blockchains-work.md:97]: Yes it's normal

**Review by yurikoinaba — APPROVED** [review]

**Comment by CryptoManion** [comment]
New PR made

**Pull request closed without merging**